docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+.. _connecting-crm:
+
+Getting Started on customer integration
+#######################################
+
+This getting started manual assumes a vendor sells services to its customers.
+The vendor manages the presence of those customers on the Nuts network through the Nuts registry.
+It's very likely the vendor has software to manage customer environments. We'll call it a *CRM* where the *customers* part relates to organizations and not people.
+This does not mean that a stand-alone installation isn't supported. In that case the vendor and organization are the same.
+
+The Nuts registry enables service discovery for organizations. The registry identifies organizations through their :ref:`DID <did>`.
+The DIDs are new and do not match the internal identifiers known to the CRM.
+This connection has to be made in order for the CRM to make adjustments to DID Documents and issue Verifiable Credentials.
+
+All APIs used in the following chapters are documented at the :ref:`API <nuts-node-api>` page.
+Open API Spec files are available for generating client code.
+
+Vendor integration
+******************
+
+As a vendor, you're in power of:
+
+- running a Nuts node
+- handling organization key material
+- updating the organization DID Document
+- defining service endpoints
+- issuing name credentials for organizations
+- trusting other vendors
+
+The last three points require a setup where a vendor DID is created. This DID will act as the controller of all organization DID Documents.
+This will allow for reuse of service endpoints and issuing of verifiable credentials. It's easier for other vendors to trust a single issuer DID.
+
+Create and store a vendor DID
+=============================
+
+You'll have to prepare your CRM to store a DID. A DID is a string similar to:
+
+.. code-block:: text
+
+    did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9
+
+The DID we're about to create is your *vendor DID*. It will be used in the all of the next steps.
+For the API calls that will need to be made to the Nuts node, we'll use ``<internal-node-address>`` as the address where the internal API's are exposed.
+Consult the :ref:`configuration reference <nuts-node-config>` on how to configure the node address.
+
+.. code-block:: text
+
+    POST <internal-node-address>/internal/vdr/v1/did
+    {
+        "selfControl": true,
+        "assertionMethod": true,
```

Comment:
assertionMethod and capabilityInvocation are `true` by default, I think?

Reply:
I didn't want to bother here with what's default and what not, doesn't make thing more clear.